<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(0, 0, 0)</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>20</y>
     <width>511</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="searchInput">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Type to search...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="styleSheet">
             <string notr="true">background-color:rgb(255, 38, 0)</string>
            </property>
            <property name="text">
             <string>search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>All Conversations</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="currentText">
         <string>Select Filter</string>
        </property>
        <item>
         <property name="text">
          <string>Select Filter</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mentor Name</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Date</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Topic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Status</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="mentorInterviewTable">
        <property name="styleSheet">
         <string notr="true">background-color:
rgb(255, 38, 0)</string>
        </property>
        <property name="rowCount">
         <number>10</number>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column>
         <property name="text">
          <string>Mentor Name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Date</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="returnButton">
      <property name="text">
       <string>Return to Preferences Screen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
